runtimes/runtimes_test.go

```diff
@@ -30,9 +31,12 @@ func TestParseOpenWhisk(t *testing.T) {
 	converted := ConvertToMap(openwhisk)
 	println(converted["nodejs"])
 	println(converted["python"])
+	//println(converted["go"])
 	assert.Equal(t, 2, len(converted["nodejs"]), "not expected length")
 	assert.Equal(t, 2, len(converted["php"]), "not expected length")
 	assert.Equal(t, 1, len(converted["java"]), "not expected length")
 	assert.Equal(t, 6, len(converted["python"]), "not expected length")
 	assert.Equal(t, 2, len(converted["swift"]), "not expected length")
+	// cannot pass this test until it is deployed in OpenWhisk
```

Comment:
go is now part of the deployment runtimes in OprnWhisk, what do you mean?

Reply:
The test looks like it tries to parse BlueMix runtime.json, and it fails

Reply:
ok understood, leave commented out